cmake/README.md

```diff
@@ -100,7 +100,18 @@ test it on public CI and support can be broken.**
 
 There are several options that can be passed to CMake to modify how the code is built.<br>
 For all of these options and parameters you have to use `-D<Parameter_name>=<value>`.<br>
-Following a list of available options, for the full list run:
+
+For example, to generate build files including dependencies in a new subdirectory called 'build', run:
+```sh
+cmake -DBUILD_DEPS:BOOL=ON -H. -Bbuild
```

Comment:
I would prefer to use:
```sh
cmake -S. -Bbuild  -DBUILD_DEPS:BOOL=ON 
```
note: `-H` and `-B` were hidden options (i.e. never officially supported) so we shouldn't depend on it, while `-S` and `-B` were officially publicly provided since CMake 3.13 (and or-tools require at least CMake 3.14 IIRC)
src:  https://cmake.org/cmake/help/latest/release/3.13.html#command-line

Reply:
Helps to actually know vs copying things from StackOverflow! Thanks.